schemas/bridge/SCHEMA.md

```diff
@@ -0,0 +1,135 @@
+| token_amount_usd         | The amount supplied, in USD.                              | number |
+| volume_usd               | The bridged volume in USD, during this snapshot.          | number |
+| fees_usd                 | The fees collected in USD, during the snapshot period.    | number |
+| chain_id                 | The standard chain ID where the pool resides.             | number |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
+
+### Bridge Transactions
+
+List of bridge transactions.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the transaction.                         | number |
+| block_number             | The block number of the transaction.                      | number |
+| transaction_hash         | The hash of the transaction this bridge originated from.  | string |
+| source_chain_id          | The source standard chain ID.                             | number |
+| destination_chain_id     | The destination standard chain ID.                        | number |
+| volume_usd               | The volume of the transaction in USD.                     | number |
+| user_address             | The address of the user initiating the transaction.       | string |
+| solver_address           | The address of the solver (for intent-based bridges).     | string |
+| pool_address             | The address of the pool (for pool-based and mint-based bridges). | string |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
+
+### Maker Snapshot
+
+List of makers in intent-based bridge protocols.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| user_address             | The address of the maker.                                 | string |
+| amount_in_usd            | The amount received in USD, during this snapshot.         | number |
+| amount_out_usd           | The amount sent in USD, during this given snapshot.       | number |
+| fees_usd                 | The fees collected in USD, during this snapshot.          | number |
+| chain_id                 | The standard chain ID where the maker resides.            | string |
+
+### Taker Snapshot
+
+List of takers in intent-based bridge protocols.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
```

Comment:
token_address
token_amount

Reply:
Also, I updated `volume_usd` to `token_amount_usd`. Lmk if the descriptions make sense in this case, I am less familiar with intent-based bridges

Reply:
I guess I am confused if this is a volume or a balance?

Reply:
this should be volume. to get the volume in token amount

Reply:
Okay, then I am using `volume_amount` and `volume_usd`